package.json

```diff
@@ -47,7 +47,7 @@
   "devDependencies": {
     "sass": "^1.49.9",
     "entcore-css-lib": "npm:ode-csslib@4.0.12",
-    "@mdi/font": "^4.5.95",
+    "@mdi/font": "7.4.47",
```

Comment:
Est ce que tu peux expliquer en description cette monter de version ? Et check "Any dependent changes have been added to this project (must specify in Description)"